From 1de329ba0067420a4558ff140464fd3a8e6a1582 Mon Sep 17 00:00:00 2001
From: Henry Wu <Henry_Wu@quantatw.com>
Date: Thu, 19 Mar 2026 16:31:12 +0800
Subject: [PATCH] configuration: santabarbara: configure MMC as MCTPI2CTarget
 in BiosPost

Mark MMC devices as MCTPI2CTarget and set PowerState to BiosPost,
indicating they are valid after BIOS POST.

The link of the gerrit:
https://gerrit.openbmc.org/c/openbmc/entity-manager/+/88430

Signed-off-by: Henry Wu <Henry_Wu@quantatw.com>
---
 configurations/meta/santabarbara/santabarbara_rainbow.json | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configurations/meta/santabarbara/santabarbara_rainbow.json b/configurations/meta/santabarbara/santabarbara_rainbow.json
index 3cb4b66..3e353e3 100644
--- a/configurations/meta/santabarbara/santabarbara_rainbow.json
+++ b/configurations/meta/santabarbara/santabarbara_rainbow.json
@@ -20,6 +20,8 @@
             "Address": "0x21",
             "Bus": "$bus / 2 + 2 * 3 % 5 * 3 + 1 / 4 + 102",
             "Name": "MMC",
+            "Inventory": "Rainbow",
+            "PowerState": "BiosPost",
             "Type": "MCTPI2CTarget"
         }
     ],
-- 
2.53.0-Meta

